Formula/l/lsr.rb

```diff
@@ -0,0 +1,37 @@
+class Lsr < Formula
+  desc "Ls but with io_uring"
+  homepage "https://tangled.sh/@rockorager.dev/lsr"
+  url "https://tangled.sh/@rockorager.dev/lsr",
+      using:    :git,
+      tag:      "v1.0.0",
+      revision: "9bfcae0be1d3ee2db176bb8001c0f46650484249"
```

Comment:
No release tarball?

Reply:
As far as I know, it was reported 2 years ago. That is really strange point.
I will try to contact to upstream for creating tarball and discuss about this bug.